README.md

```diff
@@ -1,40 +1,75 @@
 # ssv-dkg
 
-### Build
+## Table of contents:
 
-```sh
-make install
-```
+- [ssv-dkg](#ssv-dkg)
+  - [Table of contents:](#table-of-contents)
+  - [Goal and Introduction](#goal-and-introduction)
+    - [DKG](#dkg)
+    - [DKG tool by SSV](#dkg-tool-by-ssv)
+  - [Quick-start](#quick-start)
+    - [Overview](#overview)
+    - [Build](#build)
+    - [Operator](#operator)
+      - [Start a DKG-operator](#start-a-dkg-operator)
+      - [Launch with Docker and YAML file](#launch-with-docker-and-yaml-file)
+    - [Initiator](#initiator)
+      - [Generate Initiator identity RSA key pair](#generate-initiator-identity-rsa-key-pair)
+      - [Start DKG Initiator](#start-dkg-initiator)
+      - [Launch with Docker and YAML file](#launch-with-docker-and-yaml-file-1)
+      - [`init` message fields:](#init-message-fields)
+      - [Obtaining Operators data](#obtaining-operators-data)
+  - [Architecture](#architecture)
+    - [Flow Description:](#flow-description)
+    - [Note on DKG instance management](#note-on-dkg-instance-management)
+  - [Security notes](#security-notes)
+  - [Development Notes:](#development-notes)
+    - [TODO:](#todo)
+    - [Additional:](#additional)
+    - [Flow TODO Brakedown](#flow-todo-brakedown)
+      - [Round 1](#round-1)
+      - [Round 2](#round-2)
 
-### Operators data
 
-The data of the operators (ID, IP, Pubkey) can be collected in any way, for example a central server that you can pull the data from, or a preset file where all operators data exist.
+## Goal and Introduction
 
-Information about operators can be collected at `json` file and supplied to initiator to use for a key generation.
+### DKG
 
-Operators info file example (`./examples/operators_integration.json`):
+Distributed Key Generation is a cryptographic process that aims to solve the problem of coordinating `N` parties to cryptographically sign and verify signatures without relying on Trusted Third Parties.
+The process is demonstrated to be successful in successfully computing a key pair in the presence of a number `T` attackers in a decentralized network.
```

Comment:
this is not entirely correct since we mimic the decentralized network with our initiator hub. also this sounds a bit funny - `demonstrated to be successful in successfully computing`

Reply:
Why is it not true if we mimic a decentralized network?

In case this wasn't clear, this paragraph is generically explaining/introducing DKG and the principles behind it.

Reply:
I have understood the **successful in successfully** repetition only now.
In regards to the `network` comment: this is the "generic" DKG section, the specifics of SSV's implementation are explained below